api/src/main/scala/WebServer.scala

```diff
@@ -0,0 +1,25 @@
+package org.wikiwatershed.mmw.geoprocessing
+
+import akka.actor.ActorSystem
+import akka.actor.Props
+import akka.event.Logging
+import akka.stream.ActorMaterializer
+import akka.io.IO
+import akka.http.scaladsl.Http
+import com.typesafe.config.ConfigFactory
+
+object Main {
+  implicit val system = ActorSystem("mmw-geoprocessing")
+  implicit val materializer = ActorMaterializer()
+  protected implicit val log = Logging(system, "app")
```

Comment:
From what I can tell, we've been trying to standardize on https://github.com/typesafehub/scala-logging by making use of the `LazyLogging` trait to make a `logger` val available within classes. It would be good to try and use that approach again here vs. the lower level `Logging` instance.